src/java/com/zimbra/qa/selenium/projects/ajax/tests/mail/mail/DisplayMailContent.java

```diff
@@ -40,11 +40,12 @@ public DisplayMailContent() {
 	public void DisplayHTMLMailWithHTMLPref_01() throws HarnessException {
 
 		String multilineHtmlData = null;
-	 	if (ConfigProperties.getStringProperty("browser").contains("firefox")) {
-	 	 	 multilineHtmlData = "<div><span style=\"font-size: 14pt;\"><span style=\"font-family: &quot;comic sans ms&quot;,&quot;comic sans&quot;,sans-serif;\">Number list below</span>:</span></div>";
-	 	} else {
-	 	 	 multilineHtmlData = "<div><span style=\"font-size: 14pt;\"><span style=\"font-family: &quot;comic sans ms&quot;, &quot;comic sans&quot;, sans-serif;\">Number list below</span>:</span></div>";
-	 	}
+		if (ConfigProperties.getStringProperty("browser").contains("firefox")
+				|| ConfigProperties.getStringProperty("browser").contains("edge")) {

```

Comment:
nit: instead of `contains` prefer using `equals` to be consistent with how we match in the other two changes, be consistent in how we match "edge".

Reply:
Hi Phil, thanks for your comment. We used to have msedge/edge in user config according to people's convenience so we are contains using edge.